tests/test_spectral_nomination.py

```diff
@@ -0,0 +1,137 @@
+import pytest
```

Comment:
Not sure exactly what you mean, the biggest difference as far as I can tell is the use of standard pytest decorators (e.g. `pytest.mark.parameterize` ? I guess more commenting or splitting up some of the functions would be good? Or moving helper functions outside of the actual test class?

Reply:
If you look at some of the other tests in graspologic (see test_mase or test_svd or test_match) it's usually structured such that you test the inputs, then test outputs (ie making sure the code is working as intended). For example, in the input check below it's clear that I am checking for type error, specifically for `embedder`, which expects a baseembedder type or string 'ASE' or 'LSE'
```
with pytest.Raises(TypeError):
    SpectralVertexNomination(embedder = 'hi')
```
I can't tell from looking at your code if you have this type check in your test function, because it's not clear.